ironfish-cli/src/api.ts

```diff
@@ -37,7 +37,9 @@ export class IronfishApi {
       previous_block_hash: block.previous,
       difficulty: block.difficulty,
       graffiti: block.graffiti,
-      transactions_count: 0,
+      main: block.main,
+      transactions: block.transactions,
+      transactions_count: block.transactions.length,
```

Comment:
Why are we sending this up? Isn't this just a derived value from transactions?

Reply:
Good point; changed it here and implemented the requisite change in the API.